HACKING.md

```diff
@@ -33,6 +35,37 @@ Follow the instructions to install the operator, e.g.:
 $ oc apply -f /tmp/manifests/path
 ```
 
+#### Building Remotely on the Cluster
+
+To build the operator on the remote cluster, first create a buildconfig.  Use the following command, with the appropriate strings substituted for `<username>` and `<branch>`:
+
+```
+$ oc process -f hack/buildconfig.yaml \
+     -p 'GIT_URL=https://github.com/<username>/cluster-ingress-operator.git' \
+     -p 'GIT_BRANCH=<branch>' \
+      | oc -n openshift-ingress-operator create -f -
+```
+
+Next, start a build from this buildconfig:
+
+```
+$ oc -n openshift-ingress-operator start-build cluster-ingress-operator --follow
```

Comment:
The `make build-cluster` process could start this as well

Reply:
We only need to create the buildconfig once, and then we can start arbitrarily many builds using it, so we can either have separate make targets or do some extra work to check whether the buildconfig exists first.  Would separate `make buildconfig-cluster` and `make build-cluster` targets be all right?

Reply:
Done, using two targets: `buildconfig` and `cluster-build`.